[2024-06-12T03:44:51.213+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task3 manual__2024-06-12T03:44:47.077617+00:00 [queued]>
[2024-06-12T03:44:51.218+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task3 manual__2024-06-12T03:44:47.077617+00:00 [queued]>
[2024-06-12T03:44:51.218+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-06-12T03:44:51.218+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-06-12T03:44:51.218+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-06-12T03:44:51.225+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): task3> on 2024-06-12 03:44:47.077617+00:00
[2024-06-12T03:44:51.228+0000] {standard_task_runner.py:55} INFO - Started process 1071 to run task
[2024-06-12T03:44:51.230+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dag_with_callback', 'task3', 'manual__2024-06-12T03:44:47.077617+00:00', '--job-id', '312', '--raw', '--subdir', 'DAGS_FOLDER/example_dag_with_callback.py', '--cfg-path', '/tmp/tmp9qx0qufz']
[2024-06-12T03:44:51.231+0000] {standard_task_runner.py:83} INFO - Job 312: Subtask task3
[2024-06-12T03:44:51.263+0000] {task_command.py:376} INFO - Running <TaskInstance: example_dag_with_callback.task3 manual__2024-06-12T03:44:47.077617+00:00 [running]> on host 3eba8accb13f
[2024-06-12T03:44:51.296+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_dag_with_callback
AIRFLOW_CTX_TASK_ID=task3
AIRFLOW_CTX_EXECUTION_DATE=2024-06-12T03:44:47.077617+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-12T03:44:47.077617+00:00
[2024-06-12T03:44:51.296+0000] {logging_mixin.py:120} INFO - Task 3 is completed
[2024-06-12T03:44:52.299+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-06-12T03:44:52.322+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=example_dag_with_callback, task_id=task3, execution_date=20240612T034447, start_date=20240612T034451, end_date=20240612T034452
[2024-06-12T03:44:52.360+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-06-12T03:44:52.373+0000] {local_task_job.py:273} INFO - 2 downstream tasks scheduled from follow-on schedule check
